src/OrchardCore.Modules/OrchardCore.Workflows/Events/WorkflowFaultEvent.cs

```diff
@@ -0,0 +1,85 @@
+using Microsoft.Extensions.Localization;
+using OrchardCore.Workflows.Abstractions.Models;
+using OrchardCore.Workflows.Activities;
+using OrchardCore.Workflows.Models;
+using OrchardCore.Workflows.Services;
+using OrchardCore.Workflows.ViewModels;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Workflows.Events
+{
+    public class WorkflowFaultEvent : EventActivity
+    {
+        private readonly IStringLocalizer<WorkflowFaultEvent> S;
+        private readonly IWorkflowScriptEvaluator _scriptEvaluator;
+
+        public WorkflowFaultEvent(IStringLocalizer<WorkflowFaultEvent> stringLocalizer,
+             IWorkflowScriptEvaluator scriptEvaluator
+            )
+        {
+            S = stringLocalizer;
+            _scriptEvaluator = scriptEvaluator;
+        }
```

Comment:
Add a new line after. Consider using CTRL + k + d to format your code in Visual Studio.

Reply:
```csharp
    public WorkflowFaultEvent(
        IStringLocalizer<WorkflowFaultEvent> stringLocalizer,
        IWorkflowScriptEvaluator scriptEvaluator)
    {
            S = stringLocalizer;
            _scriptEvaluator = scriptEvaluator;
    }
```

Reply:
This is the effect of my visual studio formatting shortcuts....😕
![image](https://user-images.githubusercontent.com/15613121/177051176-d3eda239-65f0-4673-8725-a9e315f567b9.png)